src/Extensions/ExtHostClient.re

```diff
@@ -298,6 +336,46 @@ let provideCompletions = (id, uri, position, client) => {
   promise;
 };
 
+let provideDecorations = (handle, uri, client) => {
+  let decodeItem =
+    fun
+    | (
+        _requestId,
+        `List([
+          `Int(_),
+          `Bool(_),
```

Comment:
I don't know what these are yet. But unless there's some actual functionality we're missing, we might as well just ignore them.